frontend/src/components/admin/UserItem.tsx

```diff
@@ -13,7 +13,7 @@ const UserItem = ({ id, name, is_admin, email, avatar_url, created_at }: IUser):
         <Avatar 
           width={32}
           height={32}
-          url={`${avatar_url}`} 
+          url={`${avatar_url === null ? 'https://i.stack.imgur.com/l60Hf.png' : avatar_url}`} 
```

Comment:
Is this the default avatar? If so, please put `https://i.stack.imgur.com/l60Hf.png` in a constant variable.